Gemfile

```diff
@@ -268,7 +268,9 @@ end
 
 group :test do
   # Use capybara to simulate user-browser interaction
-  gem("capybara")
+  gem("capybara", "~> 3.37", ">= 3.37.1")
```

Comment:
I don't know why this doesn't already pull the latest Capybara (even with gem update). I upped the version at the suggestion of the Capybara author.